File: linear_search.py
def linear_search(list, target):
    # if target in list, return index of target, else return none.
    for i in range(0, len(list)):
        if list[i] == target:
            return i
    # if not in the list
    return None


def verify(index):
    if index is not None:
        print("Target found at index: ", index)
    else:
        print("Target not found in list")


numbers = [1,2,3,4,5,6,7,8,9,10, 11, 12]

result = linear_search(numbers, 12)
verify(result)

File: linear_search.js
function linear_search(list, target) {
    // return index of target, if not found return none
    for(item in list) {
        if(item == target) {
            console.log("found! " + target)
            return target
        }
        else if(item < target) {
            console.log("the target is higher");
        }
        else if(item > target) {
            console.log("the target is lower");
        }
        else {
            console.log("Item not in list");
        }
    };
};

list = [...Array(30).keys()];
console.log(list);
linear_search(list, 23);